src/index.jsx

```diff
@@ -25,11 +25,13 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    //console.log((isNaN(window.location.pathname.split('/')[2]*1)) ? 5: window.location.pathname.split('/')[2]*1);
-    this.setState({productId: (isNaN(window.location.pathname.split('/')[2]*1)) ? 5: window.location.pathname.split('/')[2]*1}, () => this.getReviews());
```

Comment:
maybe you could set this up as an if/else block so you don't have to worry about changes when you are later on EC2?